src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -1404,6 +1409,7 @@ avtBlueprintFileFormat::PopulateDatabaseMetaData(avtDatabaseMetaData *md)
         err_oss <<  "Conduit Exception in Blueprint Plugin "
                     << "Populate Database MetaData: " << endl
                     << e.message();
+        visitTimer->StopTimer(t_pop_md,"PopulateDatabaseMetaData");
```

Comment:
I wonder if this is a coding issue we have in other parts of VisIt as well (not suggesting you address if so) because when I've looked at timer reports, I always seem to find unmatched timers

Reply:
I suspect we do have this issue elsewhere. In my case the Blueprint reader was throwing exceptions that caused some `StartTimer` calls to be unmatched making the output almost impossible to make sense of. My guess is that its rare for us to be interested in timing code when it throws exceptions.